[doc] Clean up gui.md (#62)
taichi-dev/docs.taichi.graphics: website/docs/lang/articles/misc/gui.md

        elif gui.is_pressed('d', ti.GUI.RIGHT):
            print('Go right!')

`gui.get_cursor_pos()` can return current cursor position within the window. For example:

    mouse_x, mouse_y = gui.get_cursor_pos()

`gui.fps_limit` set the FPS limit for a window. For example, to restrict FPS to be below 24, simply `gui.fps_limit = 24`. This helps reduce the overload on your hardware especially when you're using OpenGL on your integrated GPU which could make desktop slow to response.

## GUI Widgets

> Sometimes it's more intuitive to use widgets like slider, button to control program variables instead of chaotic keyboard bindings. Taichi GUI provides a set of widgets that hopefully could make variable control more intuitive:

Reviewer: ```suggestion
Sometimes it's more intuitive to use widgets like slider or button to control the program variables instead of using chaotic keyboard bindings. Taichi GUI provides a set of widgets for that reason:
```
